ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -64,13 +66,22 @@ import {
   MetaMetricsEventCategory,
   MetaMetricsEventName,
 } from '../../../../shared/constants/metametrics';
+import { hexToDecimal } from '../../../../shared/modules/conversion.utils';
 import { MetaMetricsContext } from '../../../contexts/metametrics';
+import { MergedInternalAccount } from '../../../selectors/selectors.types';
 import {
-  getCaip25PermissionsResponse,
   PermissionsRequest,
   getRequestedCaip25CaveatValue,
+  getDefaultAccounts,
+  getAllRequestedAccounts,
+  getAllRequestedChainIds,
 } from './utils';
 
+// put this here because of some circular dependency issue
+export const caipFormattedTestChains = TEST_CHAINS.map(
+  (chainId) => `eip155:${hexToDecimal(chainId)}`,
+);
```

Comment:
should park this somewhere else?

Reply:
yeah probably. Going to look for a good home for it

Reply:
pushing a commit to move it